spec-main/api-browser-window-spec.ts

```diff
@@ -1353,6 +1353,22 @@ describe('BrowserWindow module', () => {
       // Values can be 0,2,3,4, or 6. We want 6, which is RGB + Alpha
       expect(imgBuffer[25]).to.equal(6);
     });
+
+    it('does not freeze browser process when called on hidden page', async() => {
```

Comment:
Can you also fix the lint error?

```
/home/builduser/project/src/electron/spec-main/api-browser-window-spec.ts
  1357:75  error  Missing space before function parentheses  space-before-function-paren
  1363:22  error  There should be no space after this paren  space-in-parens
```

Reply:
@zcbenz we should go with this fix: https://github.com/electron/electron/pull/27883 (see my comment in that pull request).